haxepunk/graphics/tile/Tilemap.hx

```diff
@@ -497,9 +461,10 @@ class Tilemap extends Graphic
 
 	function drawTile(tile:Int, tx:Int, ty:Int, x:Float, y:Float, scx:Float, scy:Float)
 	{
-		updateTileRect(tile);
+		var region = _atlas.getRegion(tile);
+		HXP.rect.setTo(region.x, region.y, region.width, region.height);
 		_atlas.prepareTile(
-			_tile,
+			HXP.rect,
```

Comment:
I'd rather Tilemap have its own static Rectangle than use HXP.rect for this. If enough classes use HXP.rect it's hard to reason about whether any of their modifications might overlap. This has caused weird bugs in the past.

Reply:
That makes sense to me.  I'll add the _tile rect back in.